packages/graphql-server/src/types.ts

```diff
@@ -58,7 +58,7 @@ export type GetCurrentUser = (
   decoded: AuthContextPayload[0],
   raw: AuthContextPayload[1],
   req?: AuthContextPayload[2],
-) => Promise<null | Record<string, unknown> | string>
+) => Promise<null | Record<string, unknown>>
```

Comment:
Note this change here. I think this type is incorrect. Why would you ever want to return a string from getCurrentUser?